source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md

```diff
@@ -41,12 +41,20 @@ Possibilities to expose the new query through the REST API:
   ```
 
 ### Webadmin
-There will be two tables in the `Virtual Machines` sub-tab in the host details page.
-One for running VMs and the other for pinned VMs:
+The table in the `Virtual Machines` sub-tab on the host details page shows
```

Comment:
In oVirt 4.2 UI we moved away from main & sub tab terminology (even though the code still has that naming convention).

Please refer to tabs under details of a specific entity as "detail tabs", e.g. VM detail tab of host entity.

Reply:
Done